src/language-js/print/template-literal.js

```diff
@@ -39,11 +39,11 @@ function printTemplateLiteral(path, print, options) {
   }
   const parts = [];
 
-  let expressions = path.map(print, expressionsKey);
+  let printedExpressions = path.map(print, expressionsKey);
```

Comment:
How about `expressionDocs`?

Reply:
We always call doc as `printed` in other places.

Reply:
Definitely not always.

Reply:
I'm fine with the name, I'll use it.